test/RateLimiterMongo.test.js

```diff
@@ -567,4 +566,56 @@ describe('RateLimiterMongo with fixed window', function RateLimiterMongoTest() {
         done(err);
       });
   });
+
+  it('consume 1 point (driver v3)', (done) => {
+    const testKey = 'consume1v3';
+    sinon.stub(mongoClient, 'topology').value({ s: { options: { metadata: { driver: { version: '3.6' } } } } });
```

Comment:
@animir, does this make sense as a mock? I'm checking the resulting `upsertOptions` below.

Reply:
@backflip yes, this looks good